mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/search/definition/impl/PojoConstructorProjectionDefinitionMessages.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.search.definition.impl;
+
+import org.hibernate.search.util.common.logging.impl.MessageConstants;
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+/**
+ * Message bundle for constructor projections in the POJO mapper.
+ */
+@MessageBundle(projectCode = MessageConstants.PROJECT_CODE)
+public interface PojoConstructorProjectionDefinitionMessages {
```

Comment:
Better move this next to other message bundles in the `reporting.impl` package. Otherwise you'll get in trouble because of Java modules (we need such classes to be opened to the jboss-logging module, and we only do that for `reporting.impl` packages at the moment).

Reply:
Oh, right.. moved the messages and rebased to include the latest from the main branch 😃